pilot/pkg/networking/core/v1alpha3/cluster_test.go

```diff
@@ -2287,7 +2288,8 @@ func TestBuildUpstreamClusterTLSContext(t *testing.T) {
 											},
 										},
 									},
-									ResourceApiVersion: core.ApiVersion_V3,
+									ResourceApiVersion:  core.ApiVersion_V3,
```

Comment:
This will cause a blocked DNS cluster to stop envoy permanently rather than for 15s, which I think is undesired?

Reply:
This is just for TLS setting. The DNS cluster block may happen because of resolution not happening which is not impacted by this change and we do not have control over that behaviour. This https://github.com/envoyproxy/envoy/pull/10728 fixed that in Envoy. So we should be good - unless you are thinking about some thing different than this?